### ISSUE TITLE
Keyword input field has no focus style

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#706]
See https://github.com/Yoast/yoast-components/commit/52e9daeb701bc063dd87c627802471e10f0d7300

While all the fields in the meta box, for example the snippet editor ones, and the synonyms one on premium, have a proper focus style, the `KeywordField` has no focus style at all.

Focus indication is a basic a11y requirement, and [seems it was intentionally reset](https://github.com/Yoast/yoast-components/commit/52e9daeb701bc063dd87c627802471e10f0d7300) with some `!important` CSS declarations.

Also, the `hasError` class won't work because the border and outline are already reset, so the color change on the border and outline won't produce any effect.w
